pyproject.toml

```diff
@@ -23,7 +23,7 @@ dependencies = [
     'matplotlib',
     'stsci.tools>=4.0',
     'stsci.image>=2.3.4',
-    'stsci.imagestats>=1.8.1',
+    'stsci.imagestats',
```

Comment:
Do we need a max on the version, to avoid the version that caused the earlier problems?

Reply:
No.  Because the current (RC5) has stsci.imagestats set to 1.6.3, this restricts the version of the package that RC6 is allowed to use.